.github/workflows/ci.yml

```diff
@@ -268,4 +268,3 @@ jobs:
         with:
           cname: ${{ env.DOCUMENTATION_CNAME }}
           token: ${{ secrets.GITHUB_TOKEN }}
-          python-version: ${{ env.MAIN_PYTHON_VERSION }}
```

Comment:
This argument is no longer needed/supported. I noticed a build warning when this action was run as part of completing the 0.2.0 release - it otherwise does not get run which is why it had not been spotted. Fortunately, it was just a warning that did not interfere with the doc upload for the release in any way.